modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -3551,6 +3621,46 @@ private ObjectAction _createObjectAction(String objectActionName) {
 		return objectAction;
 	}
 
+	private int _getObjectEntryVersionsCountByObjectDefinition(
+		long objectDefinitionId) {
+
+		return _objectEntryVersionLocalService.dslQueryCount(
+			DSLQueryFactoryUtil.countDistinct(
```

Comment:
Since the quantity doesn't matter to you, it's better to just use count without distinct, as it will perform better.